### PR TITLE
Add stand-alone FSM node to handle WS requests

### LINKED ISSUE #23: Add stand-alone FSM node to handle WS requests
Because the gateway is load-balanced it's hard for the clients to use the FSM because of the cookie requirements, even more it's hard to relay 2 different clients to the same instance because they need to share the cookie as well

### PR DESCRIPTION
**Pull request by IvayloPapazov**
close #23

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -76,11 +76,46 @@ module "nodes_api_uat_singapore" {
...
 }
 
+module "nodes_api_uat_singapore_channels" {
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v2.3.1"
+  env               = "api_uat"
+  envid             = "api_uat"
+  bootstrap_version = var.bootstrap_version
+  vault_role        = "ae-node"
+  vault_addr        = var.vault_addr
+  subnets           = module.nodes_api_uat_singapore.subnets
```
Comment (venimus) [inline main.tf]: I guess this should be in outputs

Comment (IvayloPapazov) [reply main.tf]: https://github.com/aeternity/terraform-aws-aenode-deploy/blob/master/outputs.tf#L9

**Review by venimus — APPROVED** [review]

**Review by dincho — CHANGES_REQUESTED** [review]
We should run the FSM in Europe for now as most of it's users would be located here

**Comment by IvayloPapazov** [comment]
> We should run the FSM in Europe for now as most of it's users would be located here

5a11fcd

**Review by dincho — APPROVED** [review]

**Pull request merged**